dll/win32/advpack/files.c

```diff
@@ -381,7 +383,9 @@ static HRESULT DELNODE_recurse_dirtree(LPWSTR fname, DWORD flags)
         {
 #ifdef __REACTOS__
 deleteinitialdirectory:
+#endif
             TRACE("%s: directory\n", debugstr_w(fname));
```

Comment:
@SergeGautherie If this TRACE is indeed part of Wine code itself too, could you then change the diff as follows? (see next suggestion)

Reply:
It is from upstream.